### PR TITLE
feat: ui improvements

### PR DESCRIPTION
**Pull request by pietrocomelli**
- Edit CSS
- Updated views
- Added Map link

### CODE REVIEW

File: public/css/hardware.css [hunk 1]
```diff
@@ -52,7 +52,7 @@ body {
     padding: 0;
   }
   
-  li {
+  #rows {
```
Comment (gsanchietti) [inline public/css/hardware.css:55]: Use a class, not an id.
```suggestion
  .rows {
```

File: resources/views/hardware.blade.php [hunk 2]
```diff
@@ -42,7 +55,7 @@
                 <!-- Looping through hardware details -->
                 @foreach ($group['rows'] as $row)
                     <!-- Displaying hardware details and occurrences -->
-                    <li>{{ $row }} <span>({{ $group['occurrences'][$row] }})</span></li> 
+                    <li id="rows">{{ $row }} <span>({{ $group['occurrences'][$row] }})</span></li>
```
Comment (gsanchietti) [inline resources/views/hardware.blade.php:58]: The should not be duplicated:
```suggestion
                    <li class="rows">{{ $row }} <span>({{ $group['occurrences'][$row] }})</span></li> 
```

**Review by gsanchietti — APPROVED** [review]

File: routes/web.php [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
+})->name('map');
```
Comment (gsanchietti) [inline routes/web.php:20]: @Tbaile can you check if this is correct?

Comment (Tbaile) [reply routes/web.php:20]: You can name a route in any way you want, it seems fine since it's used in `resources/views/select.blade.php`

**Pull request merged**